vercel/provider.go

```diff
@@ -57,6 +57,7 @@ func (p *vercelProvider) Resources(_ context.Context) []func() resource.Resource
 		newEdgeConfigResource,
 		newEdgeConfigSchemaResource,
 		newEdgeConfigTokenResource,
+		newFirewallConfigResource,
```

Comment:
is there value in having a DataSource too? (probably not?)

Reply:
Yea ideally I'd want to have a couple separate datasources that you can actually use within the config resource to avoid this thing getting too out of control, but haven't quite spent the time to look into it.  not sure if it makes too much sense for datasource of the firewall config